### PR TITLE
use imagestream for all the s2i-* cts

### PR DESCRIPTION
**Pull request by pradeepitm12**
this change build image for all the s2i-* based clustertasks
now we are using imagestream from internal registry
image-registry.openshift-image-registry.svc:5000/openshift/<runtime>:
tag

the above change helps to remove the complexity of MAJOR_VERSION
and MINOR_VERSION and gives flexibility to add only one param
VERSION which is nothing but the tags available for the runtime

Signed-off-by: Pradeep Kumar <pradkuma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: pkg/reconciler/openshift/tektonaddon/pipelinetemplates/pipelinetemplates.go [hunk 1]
```diff
@@ -34,33 +35,30 @@ type pipeline struct {
...
-		"s2i-ruby":     {Runtime: "ruby", MinorVersion: "$(params.MINOR_VERSION)", SupportedVersions: "[2.7,2.6,2.5]", Default: "7"},
-		"buildah":      {},
+		"s2i-dotnet": {Runtime: "dotnet", Version: "$(params.VERSION)", Default: "3.1"},
+		"s2i-go":     {Runtime: "golang", Version: "$(params.VERSION)", Default: "1.13.4-ubi8"},
+		"s2i-java":   {Runtime: "java", Version: "$(params.VERSION)", Default: "11"},
+		"s2i-nodejs": {Runtime: "nodejs", Version: "$(params.VERSION)", Default: "12"},
+		"s2i-perl":   {Runtime: "perl", Version: "$(params.VERSION)", Default: "5.30"},
+		"s2i-php":    {Runtime: "php", Version: "$(params.VERSION)", Default: "7.3"},
+		"s2i-python": {Runtime: "python", Version: "$(params.VERSION)", Default: "3.8"},
+		"s2i-ruby":   {Runtime: "ruby", Version: "$(params.VERSION)", Default: "2.7"},
```
Comment (karthikjeeyar) [inline pkg/reconciler/openshift/tektonaddon/pipelinetemplates/pipelinetemplates.go:60]: Shall we update the default values here, now that all the images has ubi7/ubi8 based tags, WDYT ?.

Comment (pradeepitm12) [reply pkg/reconciler/openshift/tektonaddon/pipelinetemplates/pipelinetemplates.go:60]: Yep it makes sense, will update it to ubi8 based latest tags.
Thank you

File: pkg/reconciler/openshift/tektonaddon/pipelinetemplates/pipelinetemplates.go [hunk 2]
```diff
@@ -34,33 +35,30 @@ type pipeline struct {
 }
 
 type RuntimeSpec struct {
-	Runtime           string
-	Version           string
-	MinorVersion      string
-	SupportedVersions string
-	Default           string
+	Runtime      string
+	Version      string
```
Comment (nikhil-thomas) [inline pkg/reconciler/openshift/tektonaddon/pipelinetemplates/pipelinetemplates.go:39]: can we remove this `minorVersion` field as well

**Pull request merged**